### PR TITLE
Wrote and built the documentation

### LINKED ISSUE #37: document: calculate_percentage_change function

### LINKED ISSUE #38: document: review and merge all documents

### PR DESCRIPTION
**Pull request by NikitaShymberg**
We should first merge #47 and then re-build the documentation before merging this PR so that the example.ipynb is rendered properly.

In particular, @veerupandey could you please review the docs for your function to make sure that I've understood it correctly? Feel free to add anything you think might help clarify things!

This PR fixes #38.

### CODE REVIEW

**Comment by veerupandey** [comment]
Ready for review.

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "features_creator"
-version = "0.1.4"
+version = "1.0.0"
```
Comment (SonQBChau) [inline pyproject.toml:3]: Why do we bump it to 1.0.0 for documentation?

File: README.md [hunk 2]
```diff
@@ -41,19 +41,43 @@ $ pip install git+https://github.com/UBC-MDS/features_creator
...
+df["std_monthly_data_usage"] = calculate_standard_deviation(example_data, "data_usage")
+
+# Calculate average across time periods
+df["avg_monthly_data_usage"] = calculate_average(example_data, "data_usage")
+
+# Calculate percentage change 2 months over 2 months
+df["percent_change_data_usage"] = calculate_percentage_change(
+    example_data, "data_usage", compare_period=(2, 2)
+)
+```
```
Comment (SonQBChau) [inline README.md:95]: I haven't had a chance to review  the notebook example, but I think it's nice to append all the array into the df and display it at the end

Comment (NikitaShymberg) [reply README.md:95]: I think that's a good idea too, unfortunately the dataframe gets too wide and doesn't display nicely in the html docs :(

File: pyproject.toml (hunk 1 above)
Comment (NikitaShymberg) [reply pyproject.toml:3]: As discussed during the meeting, 1.0.0 is the correct version for this

**Pull request merged**